__init__.py

```
@@ -330,13 +336,55 @@
+
+        self.acknowledge()
+        return True
+
     @staticmethod
-    def close_window(window_id, wmctrl="wmctrl"):
+    def switch_window(window_id, wmctrl="wmctrl") -> bool:
+        try:
+            result = subprocess.run([wmctrl, '-iR', window_id])
+            if result.returncode == 0:
+                return True
+        except Exception as e:
```

Ruff

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)